easy_my_coop_website/views/subscription_template.xml

```diff
@@ -381,12 +403,12 @@
 
                                 <div
                                     t-attf-class="form-group #{error and 'zip_code' in error and 'has-error' or ''}"
+                                    name="zip_code_city_container" 
                                 >
                                     <label
                                         class="col-md-3 col-sm-4 control-label"
                                         for="zip_code"
                                     >City
-                                    </label>
```

Comment:
:thinking: Why are we removing this label close node?

Reply:
It was a mistake, I have restored it.
Thanks!